source/Calamari.Tests/Fixtures/Integration/FileSystem/CalamariPhysicalFileSystemFixture.cs

```diff
@@ -366,30 +366,6 @@ public void EnumerateFilesWithGlobShouldIgnoreGroups(string directory, string gl
 
         }
 
-        [Test]
-        [Category(TestCategory.CompatibleOS.OnlyWindows)]
-        public void LongFilePathsShouldWork()
-        {
-            var paths = new Stack<string>();
-            var path = rootPath;
-
-            for (var i = 0; i <= 15; i++)
-            {
-                path += @"\ZZZZabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz";
-                fileSystem.EnsureDirectoryExists(path);
-                paths.Push(path);
-            }
-
-            fileSystem.OverwriteFile("Some sample text", path + @"\test.txt");
-            fileSystem.DeleteFile(path + @"\test.txt");
-
-            while (paths.Any())
-            {
-                var pathToRemove = paths.Pop();
-                fileSystem.DeleteDirectory(pathToRemove);
-            }
-        }
-
```

Comment:
It's not possible to test this with XUnit because we need to be able to set the Application Manifest (which is not possible for a unit test).

Reply:
This will work "by default" now that we have a minimum support version of net462 and we already have the "LongFileSupport" in the Calamari manifest file.